src/Compilers/CSharp/Test/Semantic/Semantics/NullCoalescingAssignmentTests.cs

```diff
@@ -0,0 +1,1408 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.UnitTests;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.Semantics
```

Comment:
>namespace [](start = 0, length = 9)

Test with C# 7.3 vs. 8.0

Reply:
There's already a parser test for C# 7.3 vs 8.0. I'm not sure what an additional binder test would be trying to test?

---
In reply to: [200779238](https://github.com/dotnet/roslyn/pull/28253#discussion_r200779238) [](ancestors = 200779238)

Reply:
Parsing test is good. Thx

---
In reply to: [200789687](https://github.com/dotnet/roslyn/pull/28253#discussion_r200789687) [](ancestors = 200789687,200779238)